Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs

```diff
@@ -1085,17 +1088,17 @@ private static async Task ProcessContinuousButtonPressedAsync (MouseFlags mouseF
 	{
 		while (isButtonPressed) {
 			await Task.Delay (100);
-			//var me = new MouseEvent () {
-			//	X = point.X,
-			//	Y = point.Y,
-			//	Flags = mouseFlag
-			//};
+			var me = new MouseEvent () {
```

Comment:
`me` does not appear to be referenced below. What am I missing?

Reply:
Before it was using always the same location passed on the parameter which is called when the flag is only the button pressed without the the report mouse move. When the mouse with the button pressed is moving the task method isn't called and thus the passed parameter hasn't the current location and the `move_point` field need to be used.

Reply:
I don't see where `me` is referenced. Can you show me?

Reply:
I see your point, sorry. It really is not used. I'll fix that. The fixed was on updated the point variable.

Reply:
Done. Thanks.